ui/src/app/admin/_components/personTable.tsx

```diff
@@ -0,0 +1,284 @@
+        const numSelected = table.getSelectedRowModel().rows.length;
+        const arr = [];
+        let i;
+        for (i = 0; i < numSelected; i++) {
+            const original = table.getSelectedRowModel().rows[i].original;
+            if (original.inOwner) arr.push(original.path + ':owners');
+            if (original.inInclude) arr.push(original.path + ':include');
+            if (original.inExclude) arr.push(original.path + ':exclude');
+        }
+        await removeFromGroups(searchUid, arr);
+        setRowSelection({});
+        router.refresh();
+    };
+
+    const table = useReactTable({
+        columns: personTableColumns,
+        data: groupingsInfo,
+        getCoreRowModel: getCoreRowModel(),
+        getPaginationRowModel: getPaginationRowModel(),
+        getFilteredRowModel: getFilteredRowModel(),
+        getSortedRowModel: getSortedRowModel(),
+        state: { globalFilter, sorting, rowSelection },
+        initialState: { pagination: { pageSize } },
+        onGlobalFilterChange: setGlobalFilter,
+        onSortingChange: setSorting,
+        onRowSelectionChange: setRowSelection
+    });
+
+    return (
+        <>
+            <OwnersModal open={modalOpen} close={close} modalData={modalData} />
+            <div className="flex flex-col md:flex-row md:justify-between pt-1 mb-1">
+                <div className="flex items-center">
+                    <h1 className="text-[2rem] font-medium text-text-color md:justify-start pt-3">Manage Person</h1>
+                    <p className="text-[1.2rem] font-medium text-text-color md:justify-end pt-5 ps-2">
+                        {userInfo === undefined
+                            ? ''
+                            : '(' + userInfo.name + ', ' + userInfo.uid + ', ' + userInfo.uhUuid + ')'}
+                    </p>
+                </div>
+                <div className="flex flex-col md:flex-row md:justify-end md:w-72 lg-84 pt-3 mb-1">
+                    <GlobalFilter filter={globalFilter} setFilter={setGlobalFilter} />
+                </div>
+            </div>
+            <div className="flex flex-col md:flex-row md:justify-between items-center mb-4">
+                <SearchInput />
+                <label>
+                    <div>
+                        Check All
+                        <input
+                            className="mx-2"
+                            type="checkbox"
+                            name="checkAll"
+                            checked={userInfo === undefined ? dummyBool : table.getIsAllPageRowsSelected()}
+                            onChange={
+                                userInfo === undefined
+                                    ? () => setDummyBool(!dummyBool)
+                                    : table.getToggleAllPageRowsSelectedHandler()
+                            }
+                        />
+                        <Button
+                            className="rounded-[-0.25rem] rounded-r-[0.25rem]"
```

Comment:
I always thought it was weird that our button is not rounded on the left side. Let's keep the button normal and not straighten the left side, thanks.